source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -186,7 +186,15 @@ ContextConfigImpl::ContextConfigImpl(
                                                 default_max_protocol_version)),
       factory_context_(factory_context), tls_keylog_path_(config.tls_keylog().logfile_path()),
       tls_keylog_local_(config.tls_keylog().local_address_range()),
-      tls_keylog_remote_(config.tls_keylog().remote_address_range()) {
+      tls_keylog_remote_(config.tls_keylog().remote_address_range()), access_log_(nullptr) {
+  if (!config.tls_keylog().logfile_path().empty()) {
```

Comment:
I think this should probably be validated as (pseudo-code)
```
if (config.has_tls_keylog()) {
  if local range size == 0 && remote range size == 0) { throw }
}```

Reply:
will fix